website/content/en/docs/ansible/reference/watches.md

```diff
@@ -110,5 +110,16 @@ Some features can be overridden per resource via an annotation on that CR. The o
     vars:
       state: absent
 ```
+#### Example
+```YaML
+---
+- version: v1alpha1
+  group: test.example.com
+  kind: SelectorTest
+  playbook: playbooks/selector.yml
+  selector:
+    matchExpressions:
+     - {key: testLabel, operator: Exists, values: []}
+```
```

Comment:
IMO the example needs to be added under `An example Watches file:` with an explanation such as `# Simple example mapping Bar to a playbook with selectots`. Also, can you update the data of the example to more similar to the others (E.g bar foo and etc)?